spec/controllers/spree/api/affirm_controller_spec.rb

```diff
@@ -3,9 +3,15 @@
 module Spree
   describe Api::AffirmController, type: :controller do
     render_views
+
+    before do
+      stub_authentication!
```

Comment:
This commit should be squashed with the commit that added the tests if we're just updating it to be more correct.